course/src/main/java/org/openedx/course/presentation/unit/html/HtmlUnitViewModel.kt

```diff
@@ -27,6 +32,13 @@ class HtmlUnitViewModel(
     val isCourseUnitProgressEnabled get() = config.getCourseUIConfig().isCourseUnitProgressEnabled
     val apiHostURL get() = config.getApiHostURL()
     val cookieManager get() = edxCookieManager
+    val appUserAgent: String
+        get() {
+            val appName = resourceManager.getString(R.string.app_name)
+            val applicationId = appData.applicationId
+            val versionName = appData.versionName
+            return "http.agent $appName/$applicationId/$versionName"
```

Comment:
`http.agent` only used for back-end APIs. not for the webview. please confirm on your end too.